deepchem/models/torch_models/layers.py

```diff
@@ -2663,6 +2664,216 @@ def radial_symmetry_function(self, R: torch.Tensor, rc: torch.Tensor,
         return torch.mul(K, FC)
 
 
+class AtomicConvolutionModule(nn.Module):
```

Comment:
Split this into 2 PRs, the first for the nn.module and the second for the Torchmodel